lib/tools/apk-signing.js

```diff
@@ -152,11 +151,14 @@ apkSigningMethods.checkApkKeystoreMatch = async function (keytool, md5re, keysto
     log.debug(`keystore MD5: ${keystoreHash}`);
     let matchesKeystore = entryHash && entryHash === keystoreHash;
     log.debug(`Matches keystore? ${matchesKeystore}`);
+
+    // If we have a keystore match, stop iterating
     if (matchesKeystore) {
-      return true;
+      foundKeystoreMatch = true;
+      return false;
```

Comment:
OK, JQuery's **each** also follows the "return false" rule. Please ignore the previous comment.